ui/app/adapters/auth-method.js

```diff
@@ -23,26 +23,21 @@ export default ApplicationAdapter.extend({
     const isUnauthenticated = snapshotRecordArray?.adapterOptions?.unauthenticated;
     // sys/internal/ui/mounts returns the actual value of the system TTL
     // instead of '0' which just indicates the mount is using system defaults
-    const useMountsEndpoint = snapshotRecordArray?.adapterOptions?.useMountsEndpoint;
```

Comment:
reverted changes made by https://github.com/hashicorp/vault/pull/26663/files#diff-50667c363b9be9067473f2acbfbf219e7e13c9c8c6d2232ce971e4205e7932cf now that we're using a separate method